integration-tests/src/test_loop/utils/network.rs

```diff
@@ -7,9 +7,9 @@ use std::sync::{Arc, Mutex};
 
 type DropChunkCondition = Box<dyn Fn(ShardChunkHeader) -> bool>;
 
-/// Handler to drop all network messages relevant to chunk body, based on
+/// Handler to drop all endorsement messages relevant to chunk body, based on
 /// `drop_chunks_condition` result.
-pub fn partial_encoded_chunks_dropper(
+pub fn missing_chunks_endorsement_dropper(
```

Comment:
I suggest to rename this to `chunk_endorsement_dropper_by_hash` and the function below to `chunk_endorsement_dropper_by_account`, because there is already an existing dropper with different purpose.